### PR TITLE
Feature/heat map

### PR DESCRIPTION
**Pull request by GiftLanga**
Refactored the map component
Added the heat map feature
integrated to the backend

### CODE REVIEW

File: waterlog/src/Components/Map.js [hunk 1]
```diff
@@ -1,148 +1,95 @@
...
     this.props.fetchMapsData();
+    this.props.fetchHeatMapsData();
   }
 
   constructor(props) {
     super(props);
     this.state = {
       lat: -25.783425,
-			lng: 28.336046,
+      lng: 28.336046,
```
Comment (FredJEhlers) [inline waterlog/src/Components/Map.js]: Will it not be easier to have these in constants?

File: waterlog/src/utils.js [hunk 2]
```diff
@@ -1,46 +1,160 @@
...
+    case 'high':
+      return maxIntensity;
+    case 'medium':
+      return Math.ceil(maxIntensity / 2);
+    case 'low':
+      return 1;
+    default:
+      return 0;
+  }
+}
```
Comment (FredJEhlers) [inline waterlog/src/utils.js]: Empty line EOF

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/Map.js (hunk 1 above)
Comment (GiftLanga) [reply waterlog/src/Components/Map.js]: I am setting default values, do I still need to put them in variables

Comment (GiftLanga) [reply waterlog/src/Components/Map.js]: I moved the values to a constant

File: waterlog/src/utils.js (hunk 2 above)
Comment (GiftLanga) [reply waterlog/src/utils.js]: added end of line

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**